src/UI/Badge.elm

```diff
@@ -177,6 +178,13 @@ successDark content =
     Badge { content = content, tone = ToneSuccess, brightness = Dark }
 
 
+withTone : (String -> Badge) -> Badge -> Badge
+withTone builder badge =
```

Comment:
why not `withTone builder (Badge {content})`?

Reply:
right I forgot I could do that